공장에서는 시제품의 농도가 60이라고 주장하며 품질관리팀에서 10개의 샘플을 뽑았다. 유의수준 5%에서 다음을 검정하시오.

1. 연구가설 , 귀무가설을 작성하시오.

H0 : 시제품의 농도의 평균은 60이다. 

H1 : 시제품의 농도의 평균은 60이 아니다. 

2. 유효한 샘플의 수를 계산하시오.

3. 검정통계량을 구하고 연구가설 채택여부를 작성하시오.

In [1]:
import pandas as pd
import numpy as np

data = [52,50,62,75,26,45,62,35,57,14]
mu = 60

정규성 검정 

H0 : 데이터는 정규분포를 따른다. 

H1 : 정규분포를 따르지 않는다. 


In [2]:
import scipy.stats as stats
from scipy.stats import shapiro

shapiro(data)

ShapiroResult(statistic=0.9657644629478455, pvalue=0.8490405082702637)

p-value값이 0.85로 귀무가설을 기각하지 않는다. 따라서 데이터는 정규분포를 따른다고 판단하였다. 

In [3]:
from scipy.stats import ttest_1samp
stats.ttest_1samp(data,popmean=mu)

Ttest_1sampResult(statistic=-2.099292080780179, pvalue=0.06519313277276138)

검정통계량은 -2.1


p-value값은 0.065로 유의수준 0.05보다 크게 나와 귀무가설을 기각하지 않는다. 따라서 시제품의 평균은 60이라고 할 수 있다.